core/src/main/codegen/templates/Parser.jj

```diff
@@ -6175,12 +6175,21 @@ SqlNode BuiltinFunctionCall() :
         // Expression(ExprContext.ACCEPT_SUB_QUERY), but that doesn't work
         // because it matches the other kind of IN.
         e = AtomicRowExpression() { args = startList(e); }
-        <IN>
-        e = Expression(ExprContext.ACCEPT_SUB_QUERY) { args.add(e);}
-        [
-            <FROM>
-            e = Expression(ExprContext.ACCEPT_SUB_QUERY) { args.add(e); }
-        ]
+        (
+            <IN>
+            e = Expression(ExprContext.ACCEPT_SUB_QUERY) { args.add(e);}
+            [
+                <FROM>
+                e = Expression(ExprContext.ACCEPT_SUB_QUERY) { args.add(e); }
+            ]
+        |
+            <COMMA>
+            e = Expression(ExprContext.ACCEPT_SUB_QUERY) { args.add(e);}
+            [
+                <COMMA>
+                e = Expression(ExprContext.ACCEPT_SUB_QUERY) { args.add(e); }
+            ]
+        )
```

Comment:
I think this is equivalent to:


```
        (
            <IN>
            e = Expression(ExprContext.ACCEPT_SUB_QUERY) { args.add(e);}
        (
            [
                <FROM>
                e = Expression(ExprContext.ACCEPT_SUB_QUERY) { args.add(e); }
            ]
        |
            <COMMA>
        )
            e = Expression(ExprContext.ACCEPT_SUB_QUERY) { args.add(e);}
            [
                <COMMA>
                e = Expression(ExprContext.ACCEPT_SUB_QUERY) { args.add(e); }
            ]
        )
```

...which I don't think is the desired result. Note, I'm not totally sure about this, I may be wrong, but even if I am wrong, can you add the appropriate parens for saftey/readability?

Reply:
Nvm, I'm right, I pushed an example test.

Reply:
nvm, im wrong, approving